lib/datasets/hepatitis.rb

```diff
@@ -0,0 +1,88 @@
+require "csv"
+
+require_relative "dataset"
+
+module Datasets
+  class Hepatitis < Dataset
+    Record = Struct.new(
+      :Class,
```

Comment:
Could you use lower case for field name?
Because Ruby uses lower case for name normally.

Reply:
thank you.
I will fix pointed matter.